src/templates/BlogPost/components/Article/Article.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './Article.module.scss';
+import './wordpress-theme.scss';
```

Comment:
can we move it on a global level? to have only modules in components/scenes folders?